src/bin/owi.ml

```diff
@@ -96,6 +96,18 @@ let fail_mode =
         ; (Assertion_only, info [ "fail-on-assertion-only" ] ~doc:assert_doc)
         ] )
 
+let exploration_strategy =
+  let fifo_doc = "explore the tree in fifo order" in
+  let lifo_doc = "explore the tree in lifo order" in
+  let random_doc = "explore the tree randomly" in
+  Arg.(
+    value
+    & vflag Cmd_sym.Random
```

Comment:
I don't think we want the `Random` strategy by default! I would rather use `LIFO` which is likely to be the best one. Or `FIFO` which is similar to what we had before, and thus should not change the output of too many tests for now (and we can change the default later).